spec/requests/api/internal/feedback_sessions_spec.rb

```diff
@@ -45,4 +45,48 @@ def perform
       it { expect(response.status).to eq(401) }
     end
   end
+
+  describe 'POST /create' do
+    let!(:receiver_user) { create(:user) }
+    let!(:provider_user) { create(:user) }
+    let!(:feedback_session_date) { Time.now.strftime("%Y-%m-%d") }
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Rails/TimeZone: Do not use Time.now without zone. Use one of Time.zone.now, Time.current, Time.now.in_time_zone, Time.now.utc, Time.now.getlocal, Time.now.xmlschema, Time.now.iso8601, Time.now.jisx0301, Time.now.rfc3339, Time.now.httpdate, Time.now.to_i, Time.now.to_f instead.